### PR TITLE
#497 add ghost and outline state for default color button

### PR DESCRIPTION
**Pull request by TheHadiAhmadi**

### CODE REVIEW

**Comment by TheHadiAhmadi** [comment]
@abdolian `text-shadow: 0 0 2px` section is not in flowbite. but I think it is good to have for hover state of our ghost buttons

File: src/FluentCMS.Web.UI/static/scss/components/_button.scss [hunk 1]
```diff
@@ -96,6 +96,8 @@ $prefix-button: $prefix + "button";
 
         &:hover {
             @apply text-#{$value}-800;
+			text-shadow: 0 0 2px;
```
Comment (babakhani) [inline src/FluentCMS.Web.UI/static/scss/components/_button.scss:99]: Couldn’t we use Tailwind classes to do that?

Comment (babakhani) [reply src/FluentCMS.Web.UI/static/scss/components/_button.scss:99]: I think if it’s not part of the available options in Tailwind, we should drop it.

Comment (TheHadiAhmadi) [reply src/FluentCMS.Web.UI/static/scss/components/_button.scss]: Now I removed this and fixed problem without using text-shadow.

**Review by babakhani — APPROVED** [review]

**Pull request merged**